support/build-doc-bundle.py

```diff
@@ -171,55 +190,51 @@ def build_index_doc(configs: list[ExampleDocBuildConfig]) -> bytes:
     return INDEX_TEMPLATE.render({"examples": sorted_configs}).encode("utf-8")
 
 
+def load_config(config_file: pathlib.Path) -> ExampleDocBuildConfig:
+    with open(config_file, "rb") as fp:
+        raw_config_data = yaml.load(fp, Loader=yaml.Loader)
```

Comment:
Can `.load()` be replaced with `.safe_load()`?

```suggestion
    raw_config_data = yaml.safe_load(config_file.read_text(), Loader=yaml.Loader)
```